src/prefect/_internal/schemas/validators.py

```diff
@@ -216,3 +234,112 @@ def reconcile_schedules(cls, values):
         cls._validate_schedule(schedule.schedule)
 
     return values
+
+
+def interval_schedule_must_be_positive(v: datetime.timedelta) -> datetime.timedelta:
+    if v.total_seconds() <= 0:
+        raise ValueError("The interval must be positive")
+    return v
+
+
+def default_anchor_date(v: DateTimeTZ) -> DateTimeTZ:
+    if v is None:
+        return pendulum.now("UTC")
+    return pendulum.instance(v)
+
+
+def get_valid_timezones(v: str) -> Tuple[str, ...]:
+    # pendulum.tz.timezones is a callable in 3.0 and above
+    # https://github.com/PrefectHQ/prefect/issues/11619
+    if callable(pendulum.tz.timezones):
+        return pendulum.tz.timezones()
+    else:
+        return pendulum.tz.timezones
+
+
+def validate_rrule_timezone(v: str) -> str:
+    """
+    Validate that the provided timezone is a valid IANA timezone.
+
+    Unfortunately this list is slightly different from the list of valid
+    timezones in pendulum that we use for cron and interval timezone validation.
+    """
+    from prefect._internal.pytz import HAS_PYTZ
+
+    if HAS_PYTZ:
+        import pytz
+    else:
+        from prefect._internal import pytz
+
+    if v and v not in pytz.all_timezones_set:
+        raise ValueError(f'Invalid timezone: "{v}"')
+    elif v is None:
+        return "UTC"
+    return v
+
+
+def validate_timezone(v: str, timezones: list) -> str:
+    if v and v not in timezones:
+        raise ValueError(
+            f'Invalid timezone: "{v}" (specify in IANA tzdata format, for example,'
+            " America/New_York)"
+        )
+    return v
+
+
+def default_timezone(v: str, values: dict) -> str:
+    timezones = get_valid_timezones(v)
+
+    if v is not None:
+        return validate_timezone(v, timezones)
```

Comment:
We use two different ways to retrieve timezones, using `pytz` or `pendulum` libraries. Unfortunately, the lists of tzs are different and it's possibly a breaking change to modify one or the other.

But just wanted to raise the awareness there